marketdata/rest_test.go

```diff
@@ -81,6 +161,176 @@ func TestTradesWithoutGzip(t *testing.T) {
+	secondResp := `{"quotes":[{"t":"2021-10-04T18:00:14.180608Z","ax":"N","ap":143.71,"as":1,"bx":"U","bp":143.69,"bs":3,"c":["R"],"z":"A"},{"t":"2021-10-04T18:00:14.210488488Z","ax":"N","ap":143.71,"as":1,"bx":"T","bp":143.69,"bs":2,"c":["R"],"z":"A"}],"symbol":"IBM","next_page_token":null}`
+	c.do = func(c *client, req *http.Request) (*http.Response, error) {
+		assert.Equal(t, "data.alpaca.markets", req.URL.Host)
+		assert.Equal(t, "/v2/stocks/IBM/quotes", req.URL.Path)
+		assert.Equal(t, "2021-10-04T18:00:14Z", req.URL.Query().Get("start"))
+		assert.Equal(t, "2021-10-04T18:00:15Z", req.URL.Query().Get("end"))
+		assert.Equal(t, "10", req.URL.Query().Get("limit"))
+		pageToken := req.URL.Query().Get("page_token")
+		resp := firstResp
+		if pageToken != "" {
+			assert.Equal(t, "SUJNfDIwMjEtMTAtMDRUMTg6MDA6MTQuMTgwNTIyMDAwWnwxMzQ0OTQ0Mw==", pageToken)
+			resp = secondResp
+		}
+		return &http.Response{
+			Body: ioutil.NopCloser(strings.NewReader(resp)),
+		}, nil
+	}
+	got, err := c.GetQuotes("IBM", GetQuotesParams{
+		Start:     time.Date(2021, 10, 4, 18, 0, 14, 0, time.UTC),
+		End:       time.Date(2021, 10, 4, 18, 0, 15, 0, time.UTC),
+		PageLimit: 10,
+	})
+	require.NoError(t, err)
+	require.Len(t, got, 12)
+	assert.True(t, got[0].Timestamp.Equal(time.Date(2021, 10, 4, 18, 00, 14, 12577217, time.UTC)))
+	assert.EqualValues(t, 143.68, got[0].BidPrice)
+	assert.EqualValues(t, 1, got[0].BidSize)
+	assert.EqualValues(t, "H", got[0].BidExchange)
+	assert.EqualValues(t, 143.71, got[0].AskPrice)
+	assert.EqualValues(t, 2, got[0].AskSize)
+	assert.EqualValues(t, "N", got[0].AskExchange)
+	assert.EqualValues(t, []string{"R"}, got[0].Conditions)
+	assert.EqualValues(t, "A", got[0].Tape)
+	assert.EqualValues(t, 143.69, got[11].BidPrice)
+}
+
+func TestGetMultiQuotes(t *testing.T) {
+	c := testClient()
+	resp := `{"quotes":{"BA":[{"t":"2021-09-15T17:00:00.010461656Z","ax":"N","ap":212.59,"as":1,"bx":"T","bp":212.56,"bs":1,"c":["R"],"z":"A"},{"t":"2021-09-15T17:00:00.010657639Z","ax":"N","ap":212.59,"as":1,"bx":"J","bp":212.56,"bs":1,"c":["R"],"z":"A"},{"t":"2021-09-15T17:00:00.184164565Z","ax":"N","ap":212.59,"as":1,"bx":"T","bp":212.57,"bs":1,"c":["R"],"z":"A"},{"t":"2021-09-15T17:00:00.18418Z","ax":"N","ap":212.59,"as":1,"bx":"N","bp":212.56,"bs":1,"c":["R"],"z":"A"},{"t":"2021-09-15T17:00:00.186067456Z","ax":"Y","ap":212.61,"as":1,"bx":"T","bp":212.57,"bs":1,"c":["R"],"z":"A"},{"t":"2021-09-15T17:00:00.186265Z","ax":"N","ap":212.64,"as":1,"bx":"T","bp":212.57,"bs":1,"c":["R"],"z":"A"}]},"next_page_token":"QkF8MjAyMS0wOS0xNVQxNzowMDowMC4xODYyNjUwMDBafEI4ODRBOUM3"}`
```

Comment:
`next_page_token` is not null? Do we test 2 things here (multi quotes with total limit)?

Reply:
Yes.